src/main/java/uk/gov/hmcts/reform/sscs/helper/PartiesNotifiedMapping.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.sscs.helper;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.SchedulingAndListingFields;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.partiesnotified.PartiesNotified;
+
+import java.util.Optional;
+
+
+public class PartiesNotifiedMapping {
+
+    private PartiesNotifiedMapping() {
+    }
+
+    public static PartiesNotified buildUpdatePartiesNotifiedPayload(HearingWrapper wrapper) {
+
+        PartiesNotified.PartiesNotifiedBuilder partiesNotifiedBuilder = PartiesNotified.builder();
+        partiesNotifiedBuilder.requestVersion(getVersionNumber(wrapper));
+        return partiesNotifiedBuilder.build();
+    }
+
+    public static String getHearingId(HearingWrapper wrapper) {
+        return Optional.of(wrapper)
```

Comment:
You want to use ofNullable in these two

Reply:
Actually These are existing functions created for Post  end points, I am just reusing them.

Reply:
Huh, I thought Optional.of would throw an NPE if anything was null in the chain

Reply:
Huh, I thought Optional.of would throw an NPE if anything was null in the chain